### PR TITLE
test the commit permission & init searchcomponent

### PR DESCRIPTION
**Pull request by vthtuy**

### CODE REVIEW

**Comment by phuongtm1911** [comment]
In server/routes/search.js, 
- add more nutrient in the response: nutrient_id=205 (carbs)
- add more params in API_FOOD: ds=Standard%20Reference&sort=r

File: .env [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
 JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
-MONGO_HOST=mongodb+srv://admin:<password>@cluster0-jx7zn.mongodb.net/mwa-prj?retryWrites=true&w=majority
\ No newline at end of file
+
+//  FOOD_API_KEY=<<key>>
+FOOD_API_KEY=ANAQtthwi1ObZhaWWV8B4VA8eGsymK5NAbLqzu2a
+
+# MONGO_HOST=mongodb+srv://admin:<<password>>@cluster0-jx7zn.mongodb.net/mwa-prj?retryWrites=true&w=majority
+
+# MONGO_HOST="mongodb://localhost:27017/mwa-prj?retryWrites=true&w=majority"
```
Comment (vnts7) [inline .env]: dont commit this

File: angular.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```
Comment (vnts7) [inline angular.json]: pls use default theme

File: .env (hunk 1 above)
Comment (vnts7) [inline .env]: dont commit this

**Pull request closed without merging**